drivers/watchdog/wdt_gecko.c

```diff
@@ -259,6 +254,13 @@ static int wdt_gecko_init(const struct device *dev)
 	CMU_ClockEnable(config->clock, true);
 #else
 	CMU_ClockSelectSet(config->clock, cmuSelect_ULFRCO);
+	/* Enable Watchdog clock. */
+	CMU_ClockEnable(cmuClock_WDOG0, true);
+#endif
+
+#if IS_ENABLED(CONFIG_WDT_DISABLE_AT_BOOT)
```

Comment:
The `IS_ENABLED` macro is meant for `if` statements in the code and not macros, so let's keep the more simple approach as before:

```suggestion
#ifdef CONFIG_WDT_DISABLE_AT_BOOT
```

Reply:
fixed